iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/LinearShuntXml.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.xml;
+
+import com.powsybl.commons.xml.XmlUtil;
+import com.powsybl.iidm.network.ShuntCompensator;
+import com.powsybl.iidm.network.ShuntCompensatorAdder;
+import com.powsybl.iidm.network.ShuntCompensatorLinearModel;
+import com.powsybl.iidm.network.VoltageLevel;
+import com.powsybl.iidm.xml.util.IidmXmlUtil;
+
+import javax.xml.stream.XMLStreamException;
+
+/**
+ * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
+ */
+public class LinearShuntXml extends AbstractConnectableXml<ShuntCompensator, ShuntCompensatorAdder, VoltageLevel> {
+
+    static final LinearShuntXml INSTANCE = new LinearShuntXml();
+
+    static final String ROOT_ELEMENT_NAME = "linearShunt";
+
+    @Override
+    protected String getRootElementName() {
+        return ROOT_ELEMENT_NAME;
```

Comment:
As I said before, it seems to me very confusing to have a different root name when just changing the model. It is still a shunt compensator.